src/js/gi/selectors/calculator.js

```diff
@@ -1083,6 +1061,29 @@ export const getCalculatedBenefits = createSelector(
       calculatedBenefits.outputs.perTerm.yellowRibbon.terms[5].visible = false;
     }
 
+    if (institutionType === 'ojt') {
+      calculatedBenefits.inputs = {
+        ...calculatedBenefits.inputs,
+        tuition: false,
+        books: false,
+        yellowRibbon: false,
+        scholarships: false,
+        tuitionAssist: false,
+        enrolled: false,
+        enrolledOld: false,
+        calendar: false,
+        working: true,
+      };
+
+      calculatedBenefits.outputs.tuitionAndFeesCharged.visible = false;
+      calculatedBenefits.outputs.giBillPaysToSchool.visible = false;
+      calculatedBenefits.outputs.yourScholarships.visible = false;
+      calculatedBenefits.outputs.outOfPocketTuition.visible = false;
+      calculatedBenefits.outputs.totalPaidToYou.visible = false;
+      calculatedBenefits.outputs.perTerm.tuitionAndFees.visible = false;
```

Comment:
Here, I'm mainly just moving the OJT "show or hide" logic block around so it takes precedence. Also added this missing line to hide the tuition fees section of the per-term breakdown.